YogaKit/Source/YGLayout.m

```diff
@@ -376,6 +376,8 @@ static void YGAttachNodesFromViewHierachy(UIView *const view)
     for (UIView *const subview in subviewsToInclude) {
       YGAttachNodesFromViewHierachy(subview);
     }
+
+    [subviewsToInclude release];
```

Comment:
This file should be ARC-enabled, so, this should automatically be included at runtime. Is there a reason to suspect this isn't being released?